run_visualizer.py

```diff
@@ -7,11 +7,16 @@
 import command_line_parser
 from visualizer import DataListener
 from visualizer import Plotter
-
+parser = argparse.ArgumentParser(formatter_class=argparse.ArgumentDefaultsHelpFormatter)
 
 setup_global_logger(level=command_line_parser.get_log_level())
+parser.add_argument('-p', '--port', type=str, default='10003', help='Port to connect to.')
+parser.add_argument('-H', '--host', type=str, default='localhost', help='Host to connect to.')
+args = parser.parse_args()
```

Comment:
I think we should just add the `-p` flag to `command_line_parser.py`. Right now you are actually creating the `ArgumentParser` twice (once by importing `command_line_parser`).

The `-p` option would be useful for all components that use `command_line_parser`, so that would actually be a worthwhile improvement.